docs/topic-guides/system-concepts/resource-pool.txt

```diff
@@ -4,22 +4,101 @@
  Resource Pool
 ###############
 
-Large-scale deep learning requires using different resources, including
-computation resources, memory, network, and storage. Typically, we start
-off prototyping our models with immediately available instance types
-with few accelerators. Once our models are mature enough, we want to
-scale up the training speed with larger instance types that have more
-powerful accelerators and high-performing processors and more instances.
-We also want to use instance types without expensive accelerators to run
-TensorBoard or any bash commands to avoid wasting accelerators.
-
-Determined provides the abstraction of physical resources for users to
-choose on their tasks. We call this abstraction *resource pools*. We can
-get the most value of our money by choosing different resource pools for
-different goals.
-
-Each resource pool is intended to map resources that are physically
-close to each other. Each resource pool can be set to use one cloud
-provider and one instance type and limited to have one region on AWS or
-zone on GCP. Each resource pool handles scheduling and scaling
+To run tasks such as experiments or notebooks, Determined needs to have
+resources (CPUs, GPUs) on which to run the tasks. However, different
+tasks have different resource requirements. For example, you may want to
+run your training on beefy V100 GPU machines, while you want your
+Tensorboards to run on cheap CPU machines with minimal resources. Given
+the cost of GPU resources, it's important to get the most value of our
+money by choosing different resource for different goals.
+
+Determined has the concept of *resource pools*, which is a set of agents
+that are identical and located physically close to each other.
+Determined allows you to configure your cluster to have multiple
+resource pools and to assign tasks to a specific resource pool, so that
+you can use different sets of resources for different tasks. Each
+resource pool handles scheduling and instance provisioning
 independently.
+
+Here are some scenarios where it can be valuable to use multiple
+resource pools:
+
+-  You create one pool, "aws-v100", that provisions p3dn.24xlarge
```

Comment:
nonblocking: quote `p3dn.24xlarge` with ``

Reply:
blocking: can you add a sentence to summarize the point for each item? For example, this summary for this item can be "use GPU for training while using CPU for TensorBoard."